src/xdp/trn_agent_xdp.c

```diff
@@ -199,11 +201,15 @@ static __inline int trn_encapsulate(struct transit_packet *pkt,
 	pkt->geneve = (void *)pkt->udp + udp_hdr_size;
 	pkt->rts_opt = (void *)&pkt->geneve->options[0];
 	pkt->scaled_ep_opt = (void *)pkt->rts_opt + sizeof(*pkt->rts_opt);
+	pkt->pod_label_value_opt = (void *)pkt->scaled_ep_opt + sizeof(*pkt->scaled_ep_opt);
```

Comment:
Has this been tested with the cases where no labels are present, both labels present, just pod label present, and just namespace label present cases?

Reply:
When there is no label, the label value = 0. The above scenarios were tested.